src/main/kotlin/build/buf/intellij/annotator/BufAnalyzeUtils.kt

```diff
@@ -185,12 +191,25 @@ object BufAnalyzeUtils {
         )
         handler.addProcessListener(object : ProcessAdapter() {
             override fun onTextAvailable(event: ProcessEvent, outputType: com.intellij.openapi.util.Key<*>) {
-                result.add(event.text.trimEnd())
+                when (outputType) {
+                    ProcessOutputType.STDOUT -> stdout.add(event.text.trimEnd())
+                }
+            }
+
+            override fun processTerminated(event: ProcessEvent) {
+                exitCode.set(event.exitCode)
             }
         }, owner)
         handler.startNotify()
-        handler.waitFor(BUF_COMMAND_EXECUTION_TIMEOUT.toMillis())
-        result.drop(1) // first line is always the CMD executed
```

Comment:
The first line is the SYSTEM output which prints the command line executed. We no longer need to do this now that we properly handle the different output types.